docs/man/docker-run.1.md

```diff
@@ -110,6 +111,14 @@ milliseconds.
 **--cidfile**=""
    Write the container ID to the file
 
+**--cpu-quota**=0
```

Comment:
Since there could be multiple cpu core, the `cpu.cfs_quota_us` could be bigger than `cpu.cfs_period_us`,  so set the `--cpu-quota` to a percentage and  then calculate it in implementation is difficult.  Maybe my documentation of the interpretation is a bit of misleading, I'll update it